src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private int index = new Random().nextInt(Colors.values().length);
```

Comment:
Don't make the index variable class-level (you should move it to method `getRandom()`. Only Random should make as class-level variable.
You can see examples in the common mistakes list:
<img width="790" alt="image" src="https://github.com/mate-academy/jv-lottery/assets/96740815/df27ec90-62a0-41bc-b4a6-fd0a3dfce74f">